templates/santa/message.html.twig

```diff
@@ -121,6 +129,13 @@
         }
       }, false);
 
+      function updateMessageLength() {
+        messageLength.innerHTML = message.value.length;
```

Comment:
We may have a difference between the PHP count and this count, specialy with emoji involved.

Maybe we should count code-point and not graphemes.

- PHP: `strlen`
- JS: `[...str].length`

See https://stackoverflow.com/a/54369605/665923

Reply:
Done